components/table/table-header.js

```diff
@@ -0,0 +1,21 @@
+import { html, nothing } from 'lit';
+import { SelectionHeader } from '../selection/selection-header.js';
+
+/**
+ * A header for table components containing a selection summary and selection actions.
+ * @slot - Responsive container using `d2l-overflow-group` for `d2l-selection-action` elements
```

Comment:
Don't think this slot description should be here? Looks like it was copied from `selection-header.js` to list as well.